go.sum

```diff
@@ -14,6 +14,8 @@ github.com/fatih/color v1.7.0 h1:DkWD4oS2D8LGGgTQ6IvwJJXSL5Vp2ffcQg58nFV38Ys=
 github.com/fatih/color v1.7.0/go.mod h1:Zm6kSWBoL9eyXnKyktHP6abPY2pDugNf5KwzbycvMj4=
 github.com/ghodss/yaml v1.0.0 h1:wQHKEahhL6wmXdzwWG11gIVCkOv05bNOh+Rxn0yngAk=
 github.com/ghodss/yaml v1.0.0/go.mod h1:4dBDuWmgqj2HViK6kFavaiC9ZROes6MMH2rRYeMEF04=
+github.com/h2so5/goback v0.0.0-20150302055225-6e210305bfc9 h1:HpHh/z38D53+rJJnOZhVGcaPImRngX18xKVma4VHBEs=
+github.com/h2so5/goback v0.0.0-20150302055225-6e210305bfc9/go.mod h1:YaAwP3MyryOTskva8K5dvyse+hrwc/OwgP4rHXcGS98=
```

Comment:
Is this change needed? I get a 404 at https://github.com/h2so5/goback

Reply:
It's the results of me doing a 
go get github.com/h2so5/goback/regexp

Does that work for you?

Reply:
`go get` on this branch works. What are we using from that library in this PR?

Reply:
Nothing anymore :smile:  I removed it again when we switched away from the complex regular expression.